stardis/io/model/MARCS.py

```diff
@@ -0,0 +1,138 @@
+import pandas as pd
+import gzip
+import re
+from dataclasses import dataclass
+
+
+@dataclass
+class MARCSModel(object):
+    """
+    Class to hold a MARCS model. Holds a dict of the meta information and a pandas dataframe of the contents.
+
+    """
+
+    meta: dict
+    data: pd.DataFrame
+
+
+def get_MARCS_meta(fpath):
+    """
+    Grabs the meta information from a gzipped MARCS model file and returns it in a python dictionary.
+    Matches the meta information and units using regex. Fails if the file does not exist or is formatted unexpectedly.
+
+    Parameters
+    ----------
+    fname : string
+            Path to model file
+
+    Returns
+    -------
+    dict : dictionary
+            parameters of file
+    """
+
+    with gzip.open(fpath, "r") as file:
```

Comment:
`rb`

Reply:
The problem was actually implicitly reading as 'rb'. Needed to use 'rt' instead for read in text mode.